src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -1012,7 +1073,7 @@ namespace OpenMS
         s_completiontime_ = xercesc::XMLString::transcode("completionTime");
         s_precision_ = xercesc::XMLString::transcode("precision");
         s_byteorder_ = xercesc::XMLString::transcode("byteOrder");
-        s_pairorder_ = xercesc::XMLString::transcode("pairOrder");
+        s_contentType_ = xercesc::XMLString::transcode("contentType");
```

Comment:
ok, so its just wonderful that this changed from mzXML 2.x to 3.0 - changing a useless attribute that is generally ignored to break backwards compatibility? well so be it, 

- do we have tests of reading 2.x and 3.x mzXML files?